2.0/contrib/src/.bashrc

```diff
@@ -1,2 +1,3 @@
 # This will make scl collection binaries work out of box.
+unset BASH_ENV
```

Comment:
@mnagy no worries I'm changing that along other changes... the problem is that if something call /bin/bash it makes it source .bashrc that might end up in endless recursion :-) I think we are safe with DB images but I will unset that just as precaution.